### PR TITLE
Add full stops in CLI help

### PR DESCRIPTION
**Pull request by jankrepl**
* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

# Description

```
dvc --help
```
The description of some commands does not end with a full stop. It is a stupid PR but for some reason this inconsistency stood out in my eyes.

### CODE REVIEW

**Review by efiop — APPROVED** [review]
Thanks!

File: dvc/command/plots.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def run(self):
 def add_parser(subparsers, parent_parser):
     PLOTS_HELP = (
         "Commands to visualize and compare plot metrics in structured files "
-        "(JSON, YAML, CSV, TSV)"
+        "(JSON, YAML, CSV, TSV)."
```
Comment (jorgeorpinel) [inline dvc/command/plots.py:98]: Many of these are not full not sentences though, so no full stop was needed.

Comment (jorgeorpinel) [reply dvc/command/plots.py:98]: In this case this one was correct without period.

File: dvc/command/unprotect.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def run(self):
 def add_parser(subparsers, parent_parser):
     UNPROTECT_HELP = (
         "Unprotect tracked files or directories (when hardlinks or symlinks "
-        "have been enabled with `dvc config cache.type`)"
+        "have been enabled with `dvc config cache.type`)."
```
Comment (jorgeorpinel) [inline dvc/command/unprotect.py:26]: But this one is a sentence indeed so `.` = 👍

File: dvc/command/plots.py (hunk 1 above)
Comment (jorgeorpinel) [reply dvc/command/plots.py:98]: Oh well it's not the only "wrong" one (very minor too), we can let it be.

**Pull request merged**